intl/msg_hash_us.h

```diff
@@ -2424,6 +2508,26 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_AUDIO_DEVICE,
    "Override the default audio device the audio driver uses. This is driver dependent."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_AUDIO_DEVICE,
+   "Override the default audio device the audio driver uses. This is driver dependent."
```

Comment:
This string is cloned from the sublabel, so shouldn't this be directly hooked to the sublabel instead?

Reply:
The first line is the same, but help text will get extended with device name examples, it is concatenated in msg_hash_us.c depending on available audio devices.

Reply:
I'm sorry, how does this make sense? What, exactly, is preventing you from concatinating MENU_ENUM_SUBLABEL_AUDIO_DEVICE to other text, or vice versa?

Reply:
I suppose it can be done, it just looks a bit off in the source code, as sublabel is not used anywhere else for such purpose. But this gave me an idea how it could be done better :)